src/onegov/org/templates/directory.pt

```diff
@@ -41,7 +41,7 @@
                             <tal:b define="custom_lead search_widget.lead(layout, entry)|nothing">
                                 <div class="list-lead">
                                     <tal:b condition="custom_lead" replace="structure custom_lead" />
-                                    <tal:b condition="not:custom_lead" replace="entry.lead" />
+                                    <tal:b condition="not:custom_lead" replace="structure python: str(entry.lead).replace('\\n', '<br>')" />
```

Comment:
This is not safe, you are making `entry.lead` vulnerable to HTML injection.
```suggestion
                                    <tal:b condition="not:custom_lead">
                                        <tal:b tal:repeat"line entry.lead.splitlines()">${line}<br tal:condition="not repeat.line.end"/></tal:b>
                                    </tal:b>
```
This is one potential of way making this safe inside pt. Another way is to add `Markup` and `escape` to the globals dictionary in our custom BaseTemplate class, so you can use `escape(entry.lead).replace('\\n', Markup('<br>')`

Reply:
Thank you for your review. It's great that you caught that.

Reply:
@Daverball By the way, I was curious that `Markup` seems to be imported (we don't have to import it in the template). From where is the import actually?